apps/wizard/app_pages/anomalist/app.py

```diff
@@ -456,6 +462,35 @@ def _change_chart_selection(df, key_table, key_selection):
     st.session_state[key_selection] = df.iloc[rows]["entity_name"].tolist()
 
 
+def _score_table(df: pd.DataFrame, df_all: pd.DataFrame, indicator_id: int) -> pd.DataFrame:
+    """Return a table of scores and other useful columns for a given indicator. Return
+    styled dataframe."""
+    # Get scores of anomalies for the same indicator
+    gf = df_all[df_all.indicator_id == indicator_id]
+    gf = gf.pivot(index=["entity_name", "year"], columns=["type"], values=["score"])["score"].add_prefix("score_")
+    gf = gf.groupby(["entity_name"]).max().reset_index()
+
+    # Select columns to display and add scores of all detectors
+    cols = ["entity_name", "score_weighted", "views"]
```

Comment:
This is very nice! Why not showing the population as well (maybe the current population, instead of the population of the anomaly year). Up to you.

Reply:
Yep, good idea. Added that.